prov/sockets/src/sock_ep.c

```diff
@@ -1794,24 +1796,31 @@ struct sock_conn *sock_ep_lookup_conn(struct sock_ep_attr *attr, fi_addr_t index
 				      union ofi_sock_ip *addr)
 {
 	int i;
-	uint16_t idx;
+	uint64_t idx;
 	char buf[8];
+	char str_addr[256];
+	size_t str_len = 256;
```

Comment:
Similar comments as above for logging the address apply to this function as well.

Reply:
done